src/SingleMovie/SingleMovie.js

```diff
@@ -21,9 +26,18 @@ class SingleMovie extends Component {
       .then(singleMovieData => {
         this.setState({ movieDetails: singleMovieData.movie  })
       })
+      .catch(error => this.setState({ error: error.message }));
   }
 
   render() {
+    if(this.state.error) {
+      return (
+        <Route render={() =>
+          <Error error={this.state.error} />
+        }
+        />
+      )
+    }
```

Comment:
This all turned out to be quite a bit easier than I expected!

Reply:
I agree. Pretty simple!